.github/workflows/build-assets.yml

```diff
@@ -330,10 +340,16 @@ jobs:
           build-args: |
```

Comment:
The "push" above should be set to false, otherwise we will end up with two pushes to jfrog.

Reply:
Done.